engine/class_modules/sc_death_knight.cpp

```diff
@@ -5073,6 +5107,14 @@ struct death_strike_heal_t : public death_knight_heal_t
   {
     death_knight_heal_t::impact( state );
 
+    if ( state -> result_total > player -> resources.max[ RESOURCE_HEALTH ] * p() -> legendary.bryndaors_might -> effectN( 2 ).percent() )
+    {
+      // Last resource cost doesn't return anything so we have to get the cost of Death Strike
+      double c = p() -> find_action( "death_strike" ) -> cost();
```

Comment:
Finding action by name during runtime doesn't sound like a great idea, even if it was like that for the Legion implementation. I think it would be better to simply store the parent death strike action somewhere and call its cost() when needed.